app/src/main/res/layout/activity_event_detail.xml

```diff
@@ -0,0 +1,128 @@
+                    android:padding="@dimen/default_margin">
+
+                    <TextView
+                        android:id="@+id/tv_company_name"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginBottom="@dimen/default_margin_mini"
+                        android:textSize="22sp"
+                        android:textStyle="bold"
+                        tools:hint="Spotify" />
+
+                    <TextView
+                        android:id="@+id/tv_company_location"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginBottom="@dimen/default_margin_mini"
+                        android:background="?attr/selectableItemBackground"
+                        android:clickable="true"
+                        android:drawableStart="@drawable/baseline_location_on_24"
+                        android:drawablePadding="@dimen/default_margin_mini"
+                        android:focusable="true"
+                        android:gravity="center_vertical"
+                        tools:hint="Cool Street 123" />
+
+                    <TextView
+                        android:id="@+id/tv_event_date"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginBottom="@dimen/default_margin_mini"
+                        android:drawableStart="@drawable/baseline_calendar_today_24"
+                        android:drawablePadding="@dimen/default_margin_mini"
+                        android:gravity="center_vertical"
+                        tools:hint="Cool Street 123" />
+                </LinearLayout>
+            </androidx.cardview.widget.CardView>
+
+            <LinearLayout
+                android:orientation="vertical"
+                android:layout_marginTop="@dimen/default_margin_small"
+                android:layout_width="match_parent"
+                android:layout_marginStart="@dimen/default_margin_mini"
+                android:padding="@dimen/default_margin_small"
+                android:clipToPadding="false"
+                android:clipChildren="false"
+                android:layout_marginEnd="@dimen/default_margin_mini"
+                android:layout_height="wrap_content">
+
+                <Button
+                    android:id="@+id/btn_pre_event"
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/button_height"
+                    android:layout_marginBottom="@dimen/default_margin_small"
+                    android:text="@string/pre_event_form"
+                    android:textColor="@android:color/black"
+                    tools:hint="Pre Event Form" />
```

Comment:
Should this value be broken out in a resource or is that not common with `hints`? :thinking:

Reply:
It's usually fine to not do that with hints since they don't need translation.